mmv1/third_party/terraform/transport/config.go.erb

```diff
@@ -1162,14 +1164,23 @@ func (c *Config) GetCredentials(clientScopes []string, initialCredentialsOnly bo
 			return *creds, nil
 		}
 
-		creds, err := transport.Creds(c.Context, option.WithCredentialsJSON([]byte(contents)), option.WithScopes(clientScopes...))
-		if err != nil {
-			return googleoauth.Credentials{}, fmt.Errorf("unable to parse credentials from '%s': %s", contents, err)
+		if c.UniverseDomain != "" && c.UniverseDomain != "googleapis.com" {
+			creds, err := transport.Creds(c.Context, option.WithCredentialsJSON([]byte(contents)), option.WithScopes(clientScopes...), internaloption.EnableJwtWithScope())
+			if err != nil {
+				return googleoauth.Credentials{}, fmt.Errorf("unable to parse credentials from '%s': %s", contents, err)
+			}
+			log.Printf("[INFO] Authenticating using configured Google JSON 'credentials'...")
+			log.Printf("[INFO]   -- Sending JwtWithScope option")
+			return *creds, nil
+		} else {
+			creds, err := transport.Creds(c.Context, option.WithCredentialsJSON([]byte(contents)), option.WithScopes(clientScopes...))
+			if err != nil {
+				return googleoauth.Credentials{}, fmt.Errorf("unable to parse credentials from '%s': %s", contents, err)
+			}
+			log.Printf("[INFO] Authenticating using configured Google JSON 'credentials'...")
+			log.Printf("[INFO]   -- Scopes: %s", clientScopes)
```

Comment:
We still want to include this line in the other path

Reply:
Added this back below "Authenticating using DefaultClient"